### ISSUE TITLE
Copy `ImportCommand` to `ImportBedrockCommand`

### ISSUE THREAD

**Issue by emhane** [paradigmxyz/reth#12476]
### Describe the feature

Copy `ImportCommand` command into a new `reth-optimism-cli` command `ImportBedrockCommand`. This is needed so a different codec can be used. Possibly makes sense to decompose the code in `reth_cli_commands::import` into smaller functions that can be imported into `reth-optimism-cli` to minimise redundancy - but not a requirement since `ImportBedrockCommand` is intended for importing the ovm chain (in the past hence and won't be effected by new eips).

blocked by https://github.com/paradigmxyz/reth/issues/12475

**Comment by emhane** [comment]
would you like to continue with this @lean-apple ? it's possible to implement it using the `BlockFileCodec`, and then in a separate pr replace the codec when https://github.com/paradigmxyz/reth/issues/12475 is solved

**Comment by emhane** [comment]
sorry @lean-apple , I saw this already exists
